inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp

```diff
@@ -900,7 +900,15 @@ void MKLDNNQuantizeNode::init() {
         IE_THROW() << "Unsupported input sizes for Quantize layer with name " << getName();
 
     if (quantizationParamsAxisesIdxs.size() == 1) {
-        axis = *quantizationParamsAxisesIdxs.begin();
+        auto axisIdx = *quantizationParamsAxisesIdxs.begin();
```

Comment:
maybe should normalize 'range' shapes (prepend 1 to rank 'data' input) before call ```initAxisIdx``` and then work with these shapes? looks like it also should fix this problem. If I missed something, correct me please

Reply:
It is possible, but in this case we will have to change the input ports for Quantize. Otherwise, an "out of range" exception will be thrown on lines 872, 875, etc.

Reply:
I mean that you should used normalized shape instead ```getParentEdgesAtPort(1)[0]->getDims()```
You shouldn't make new Constant nodes with normalized shapes

Reply:
Ok, I'll do it, thanks!

Reply:
Done, thank you!